plugins/woocommerce-blocks/assets/js/blocks/cart/inner-blocks/cart-order-summary-shipping/block.tsx

```diff
@@ -62,30 +58,18 @@ const Block = ( { className }: { className: string } ): JSX.Element | null => {
 							: __( 'Delivery', 'woocommerce' )
 					}
 					placeholder={
-						showCalculator ? (
-							<ShippingCalculatorButton
-								label={ __(
-									'Enter address to check delivery options',
-									'woocommerce'
-								) }
-							/>
-						) : (
+						! showCalculator ? (
 							<span className="wc-block-components-shipping-placeholder__value">
 								{ __(
-									'Calculated on checkout',
+									'Calculated at checkout',
```

Comment:
I think this string has changed a few times, was this in the new design? Just checking as it impacts translation availability.

Reply:
@mikejolley Yes this is from the new design. `on checkout` is kinda odd english. `at checkout` can be seen in the designs here: https://www.figma.com/design/c8VYjXqluE9vlVSE1RgJix/%5BPublic%5D-Cart-%26-Checkout-Blocks?node-id=463-4798&node-type=frame&t=svKuK4xE8ySaUzdS-0

Reply:
No worries. "Checkout" can be an action you perform or a thing, so either works I guess.